packages/smoldot-test-utils/src/index.ts

```diff
@@ -102,12 +105,13 @@ export const smoldotSpy = (responder: RpcResponder, rpcSpy: any, healthResponder
     start: async (options: SmoldotOptions): Promise<SmoldotClient> => {
       return Promise.resolve({
         terminate: () => {},
-        send_json_rpc: (rpc: string) => {
+        sendJsonRpc: (rpc: string) => {
```

Comment:
Same

Reply:
These are mocks - not the actual ones

Reply:
Even if they're mock, they're still missing a parameter?

```suggestion
        sendJsonRpc: (rpc: string, chainIndex: number) => {
```